Minor edits to `Tournament bans` (#13652)
ppy/osu-wiki: wiki/Help_centre/Tournament_bans/en.md

A tournament ban prevents a player from playing in official or officially supported tournaments. It also limits to what extent a player can staff or otherwise offer assistance in these tournaments.

> Tournament bans are very rarely handed out on their own, but generally speaking, almost all players who return under a standard appeal for an [account restriction](/wiki/Help_centre/Account_restrictions) are subject to one of at least a year in length.

Reviewer: Incorrect usage of the new comma. The previous one was fine already joining two independent clauses together.

You would only use this comma sandwich to add a clause in between, sort of as you'd do with em dashes.

Due to the high-stakes nature of tournament play, tournament bans are treated very seriously and are not subject to the kind of leniency an account restriction might ordinarily receive.

## What causes someone to receive a temporary tournament ban? {id=why-temporary}

> Any severe violation of the [community rules](/wiki/Rules) that takes place during an [officially supported tournament](/wiki/Tournaments/Official_support) is likely to cause a tournament ban, even if it would have typically only been considered a minor infringement outside of the competitive play setting.

Reviewer: Sentence is different. The new wording makes it seem like the infringement would only have been "considered", whereas the original states that the rule would've been broken outside of the competitive setting. Minor change, but:

1- Original is fine as is.
2- Changes the sentence's meaning by implying the rule break would only have been considered rather than acted on (as it should, it's a *severe* violation of the rules, and thus would've been dealt with promptly)

Reviewer 2: the original wording isn't that good since "rule violations result in infringements" doesn't make sense. the infringement would be the rule violation itself

if the point is that severe rule violations cause tournament bans even if they'd only result in silences (or otherwise less severe consequences) elsewhere, then the new version is still a little off since it's saying a "severe violation" is "considered minor" which is a contradiction, so you'd word it like this

```suggestion
Any severe violation of the [community rules](/wiki/Rules) that takes place during an [officially supported tournament](/wiki/Tournaments/Official_support) is likely to cause a tournament ban, even if it would have typically only resulted in a less severe punishment outside of the competitive play setting.
```

ofc it's ambiguous whether that's actually what was intended - just inferring based on context

in any case the premise is that a rule *was* broken, since we're talking about rule *violations* in the first place. it can then be considered minor or severe, but that's not saying anything about whether it'd be acted upon

> Any severe violation of the [community rules](/wiki/Rules) that takes place during an [officially supported tournament](/wiki/Tournaments/Official_support) is likely to cause a tournament ban, even if it would have typically only resulted in a less severe punishment outside of the competitive play setting.

Reviewer 3: ```suggestion
Any severe violation of the [community rules](/wiki/Rules) that takes place during an [officially supported tournament](/wiki/Tournaments/Official_support) is likely to cause a tournament ban, even if it would have typically only resulted in a less-severe punishment outside of the competitive play setting.
```
?

Reviewer 2: ok but no hyphen

Reviewer 3: I hate my language...

- As above, any use of third-party cheating tools or utilities to gain an advantage
- Making use of multiple accounts to enter a tournament

> - Attempting to evade a silence or other penalty by using another account

Reviewer: I don't get the change. Yes, even if semantically it is making reference to the violation, that is precisely the point. Evading the violation itself is an issue, just as much as evading a previous punishment is. Do not feel like it needs to be changed.

Reviewer 2: op is right: infringement/violation is not the same penalty/consequence, and "evading violations" doesn't semantically make sense (unless you mean avoiding making violations, which of course wouldn't be punished)

- Account sharing within/during an officially supported tournament
- Conspiring to obtain or otherwise using private information to gain or confer an advantage (such as accessing mappool information before other teams, etc.)
- Abusing positions of trust to gain or confer an advantage (manipulating schedules, unfairly changing team compositions, or disqualifying participants without cause or reasoning)
- Organising targeted abuse campaigns towards participants or organisers well outside the realm of criticism

## Can I appeal a tournament ban? {id=appeal}

> Temporary tournament bans as applied via standard appeal terms or for other reasons cannot be appealed.

Reviewer: I do not get the change, even as explained in the PR's description. Original wording looks, reads, and sounds just fine — or even better. `...via standard terms or (via) other means...` the second `via` is implicit. Moreover, if a user is Tournament Banned, they do not need to know the means. Those are transparent to them. Only the punishment concerns them.
